test/rulesets/LinearAlgebra/dense.jl

```diff
@@ -66,17 +66,20 @@ end
     end
     @testset "$f" for f in [/, \]
         rng = MersenneTwister(42)
-        for n in 3:5, m in 3:5
-            A = randn(rng, m, n)
-            B = randn(rng, m, n)
-            Ȳ = randn(rng, size(f(A, B)))
-            rrule_test(f, Ȳ, (A, randn(rng, m, n)), (B, randn(rng, m, n)))
+        @testset "Matrix" begin
+            for n in 3:5, m in 3:5
+                A = randn(rng, m, n)
+                B = randn(rng, m, n)
+                Ȳ = randn(rng, size(f(A, B)))
+                rrule_test(f, Ȳ, (A, randn(rng, m, n)), (B, randn(rng, m, n)))
```

Comment:
where are the `frule_test`s for `/` and `\`?

Reply:
I added some new testd, but this is not a PR about improving test coverage

Reply:
true

since reviewing it involves looking at most of the code in the package, good opportunity to catch things that should have issues

Reply:
I think we should just use coversage for this

Reply:
yeah, i get that -- but coverage has betrayed us before 😆

Reply:
it is the src not the test that's missing 😆 https://github.com/JuliaDiff/ChainRules.jl/issues/107